rls/src/actions/diagnostics.rs

```diff
@@ -15,7 +15,7 @@ use lsp_types::{
 };
 use rls_span::compiler::DiagnosticSpan;
 use serde_derive::Deserialize;
-use serde_json;
+
```

Comment:
We shouldn't replace unused imports with an empty line (unless it separates re-exports and regular imports or similar). This applies to more places below; could you please remove the lines altogether?